package.json

```diff
@@ -1,44 +1,52 @@
 {
-  "name": "tota11y",
-  "version": "0.1.6",
+  "name": "@khanacademy/tota11y",
+  "version": "0.2.0",
   "description": "An accessibility visualization toolkit",
-  "main": "build/tota11y.min.js",
+  "main": "dist/tota11y.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/Khan/tota11y.git"
   },
   "author": "Jordan Scales <scalesjordan@gmail.com>",
+  "contributors": [
+    "Jeff Yates <jeff.yates@alumni.manchester.ac.uk"
```

Comment:
This is cool.  I think I'll suggest this for KaTeX as well.

Reply:
Yeah, I think it should be voluntary (hence why I haven't populated it with all the folks that have had changes merged), but we should ask folks if they want to be in this list.